docs/nx/create-stacks-workspace/plugin-information.md

```diff
@@ -0,0 +1,76 @@
+---
+id: ensono-stacks-create-stacks-workspace
+title: "@ensono-stacks/create-stacks-workspace"
+sidebar_label: "@ensono-stacks/create-stacks-workspace"
+description: "Using the create-stacks-workspace with stacks!"
+keywords:
+  - NX
+  - monorepo
+  - stacks
+  - ensono
+  - "create-stacks-workspace"
+  - generator
+  - executor
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Create a new Stacks Nx workspace. Using the create-stacks-workspace script will quickly allow you to initialize a new NX workspace with the Stacks recommended plugins, packages and configuration to get started on a new project quickly.
+
+The Stacks `create-stacks-workspace` script will 
+- Run Nx [create-nx-workspace](https://nx.dev/packages/nx/documents/create-nx-workspace) command to setup a new NX workspace passing along any command line arguments provided 
+- Install any recommended plugin/generators/executors from Stacks (the exact plugins installed will depend on the preset selected see [presets](/docs/nx/create-stacks-workspace/ensono-stacks-create-stacks-workspace#presets))
+- Configure the NX workspace 
+- Execute recommended generators from Stacks to initialize the workspace/projects depending on the options provided. 
```

Comment:
Could we expand? Then link to which generators will be ran by default depending on your preset?

Reply:
I believe as it stands the only ensono plugin that is run is the Next plugin. With the empty preset and the react preset no ensono plugins are run. This does not include the workspace plugin which will always run. Will add this to documentation